swagger.yaml

```diff
@@ -0,0 +1,334 @@
+swagger: "2.0"
+info:
+  description: "An API used to customise a dataset. Once the job has been created it can then be updated with filters to apply to the dataset. Once the job is ready it is expected that the client will update the job state to start the processing. Once processing has been started the job will be placed in a locked condition as no amendments can be done."
```

Comment:
Thanks for that @gedge , that a better description for the API. I Updated the spec with it